frontend/containers/DocumentList/DocumentList.jsx

```diff
@@ -168,6 +195,28 @@ class DocumentList extends Component {
     actions.clearDocumentList()
   }
 
+  renderField(fieldName, schema, value) {
```

Comment:
@eduardoboucas I think we need to find a way to stop this being hardcoded. Perhaps not right now, but be aware that we'll have custom fields and they need to be included here